crates/rpc/src/v05/method/get_transaction_receipt.rs

```diff
@@ -244,6 +302,9 @@ pub mod types {
                     .collect(),
                 events: receipt.events.into_iter().map(Event::from).collect(),
                 revert_reason: receipt.revert_error,
+                execution_resources: receipt
+                    .execution_resources
+                    .map(ExecutionResourcesProperties::from),
```

Comment:
Side comment: I usually use `Into::into` for these cases so that struct renames have smaller diffs. Not important though.

Reply:
What I like with `From` is that you need zero effort to see the type involved, something that can be confusing with `Into`. If we don't have strong opinion on this I'll keep it like this here.